### DISCUSSION TITLE
[QoL] Add option to dismiss portable transmog vendor

### DISCUSSION THREAD

**Discussion by UnePomme** [chromiecraft/chromiecraft#6959] [Suggestions]
Current Suggestions:
**Suggestion 1** 
Add boolean value for '.transmog portable [T/F]'
  - .transmog portable summon
  - .transmog portable dismiss

**Suggestion 2**
Add "Dismiss" option in the portable tmog vendor window

**Suggestion 3**
Update '.transmog portable' to be an on/off flag
  - type .transmog portable to summon, and again to dismiss

**Comment by 55Honey** [comment]
Hi @UnePomme 
did you try to type the command again? That should despawn it.
If else fails, feel free to summon any other pet and dismiss that one.

**Reply by UnePomme** [reply]
Typing the command just resummons Mr Portable. But as long as you have a pet, summoning the pet does the trick! Ty!
